### PR TITLE
Upgrade the Stellar Python SDK to v3.0.0.

### PR DESCRIPTION
**Pull request by overcat**
Upgrade the Stellar Python SDK to v3.0.0.

This update include breaking changes, release log: https://github.com/StellarCN/py-stellar-base/releases/tag/3.0.0

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -26,10 +26,10 @@ django-model-utils = "*"
 djangorestframework = "*"
 whitenoise = "*"
 psycopg2-binary = "*"
-stellar-sdk = "~=2.10.0"
+stellar-sdk = "~=3.0.0"
 django-cors-headers = "*"
 toml = "*"
-pyjwt = "*"
+pyjwt = "==1.7.1"
```
Comment (overcat) [inline Pipfile:32]: Let's upgrade it in another independent PR.

**Comment by JakeUrban** [comment]
Hi @overcat thanks for making these changes. I'm about to take a look at the code, but I think this PR should be merged with the release of Polaris 2.0 since developers would have to update their customer integration code to use the updated SDK as well.

I've add this PR to the 2.0 milestone and created a v2.0 branch. Can you change the base branch to v2.0 in this PR?

File: Pipfile (hunk 1 above)
Comment (JakeUrban) [reply Pipfile:32]: https://github.com/stellar/django-polaris/issues/370

**Review by JakeUrban — COMMENTED** [review]
LGTM but lets change the base branch to `stellar:v2.0`

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**